pkg/config/config.go

```diff
@@ -529,6 +529,10 @@ func ReadRunModConfig(fileName string) (common.RunModConfig, error) {
 
 func ExtractReadRunModConfig(jsonData []byte) (common.RunModConfig, error) {
 	var runModConfig common.RunModConfig
+	if len(jsonData) == 0 {
```

Comment:
can you make sure, when there is no running config, we don;t error out or cause any othe issues. It is fine that the runMod config  may not have any data

Reply:
we log an error message but we dont stop server startup if there's an error related to runMod config.